examples/triton_gpt2/README.ipynb

```diff
@@ -9,17 +9,20 @@
     "\n",
```

Comment:
<p>Should we remove the <code>try</code>  / <code>except</code>  block? I.e. just leave the <code>res = ret.json()</code>  to simplify the snippet?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/SeldonIO/seldon-core/pull/3217/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='18'/>

Reply:
<p>I added this block as when running in loop (while) may cause some request  to return with empty ret =&gt; ret.json() throw exception. I believe it is a timing issue/ load on the model as without calling request in loop never happens to me.  I added the try block to make sure the script returns a sentences completion. </p><p> </p><p>I find it O.K to leave it that way - I don't think it impairs the readability / understanding. Let me know if you prefer it to be removed </p>